config/routes.rb

```diff
@@ -1,5 +1,9 @@
+require 'sidekiq/web'
+
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
+  
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
The insecurity is strong with this one! 

![](https://geekandsundry.com/wp-content/uploads/2016/11/Game-5.png)

Check [this](https://github.com/mperham/sidekiq/wiki/Monitoring) post about sidekiq monitoring! After read it, use [this solution](https://github.com/mperham/sidekiq/wiki/Monitoring) to fix this vulnerability. You will have to set two new ENV variables, `SIDEKIQ_USERNAME` and `SIDEKIQ_PASSWORD `.